framework/src/org/checkerframework/framework/util/AtmLubVisitor.java

```diff
@@ -115,14 +112,8 @@ private AnnotatedTypeMirror lubWithNull(
     private void lubPrimaryAnnotations(
             AnnotatedTypeMirror type1, AnnotatedTypeMirror type2, AnnotatedTypeMirror lub) {
         Set<? extends AnnotationMirror> lubSet;
-        if (visitingUninferedWildcard
-                        && type1.getAnnotations().size() != type1.getAnnotations().size()
```

Comment:
Although @kelloggm is the author this change, I actually made it as part of the Index Checker merge.  (Though, I thought I moved all changes to the framework out of the Index Checker branch before merging. ) Anyways, Mike is right, one of the `type1` should be `type2`.  I'll fix this in a separate PR.